INFO:torch.distributed.nn.jit.instantiator:Created a temporary directory at /tmp/tmpsfc0ucur
INFO:torch.distributed.nn.jit.instantiator:Writing /tmp/tmpsfc0ucur/_remote_module_non_scriptable.py
DEBUG:git.cmd:Popen(['git', 'version'], cwd=/home/yanmy/enrich_release, universal_newlines=False, shell=None, istream=None)
DEBUG:git.cmd:Popen(['git', 'version'], cwd=/home/yanmy/enrich_release, universal_newlines=False, shell=None, istream=None)
DEBUG:wandb.docker.auth:Trying paths: ['/home/yanmy/.docker/config.json', '/home/yanmy/.dockercfg']
DEBUG:wandb.docker.auth:No config file found
2023-12-24 00:16:14,059 - INFO - episode_0 Task: persons_3hop_MI_da f1-score: 0.9348044132397191, path: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 639, 640, 595, 596, 487, 488, 1331, 1332]
INFO:my_logger:episode_0 Task: persons_3hop_MI_da f1-score: 0.9348044132397191, path: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 639, 640, 595, 596, 487, 488, 1331, 1332]
DEBUG:filelock:Attempting to acquire lock 140668061943120 on /home/yanmy/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
DEBUG:filelock:Lock 140668061943120 acquired on /home/yanmy/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
DEBUG:filelock:Attempting to release lock 140668061943120 on /home/yanmy/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
DEBUG:filelock:Lock 140668061943120 released on /home/yanmy/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
DEBUG:filelock:Attempting to acquire lock 140668061943120 on /home/yanmy/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
DEBUG:filelock:Lock 140668061943120 acquired on /home/yanmy/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
DEBUG:filelock:Attempting to release lock 140668061943120 on /home/yanmy/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
DEBUG:filelock:Lock 140668061943120 released on /home/yanmy/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
